src/Documentation/tutorials_and_samples/tutorial_1_helloworld.md

```diff
@@ -0,0 +1,377 @@
+---
+layout: page
+title: Tutorial 1 Hello World
+---
+
+# Tutorial 1: Hello World
+
+This process recreates the same Hello World sample application available [here](https://github.com/dotnet/orleans/tree/master/Samples/2.0/HelloWorld).
+
+## Overview of the parts
+
+This application consists of a solution that contains four projects: SiloHost, OrleansClient, HelloWorld.Interfaces, and HelloWorld.Grains.
+We will create the SiloHost and OrleansClient projects as .NET Core App files and we will create the HelloWorld.Interfaces and HelloWorld.Grains projects as .NET Standard Libraries.
+Orleans comes into the picture when we add the NuGet packages to each of these four projects and replace the default code with the Orleans code provided in this tutorial.
+Finally, we will configure the dependency references among the projects before building and running the solution.
+
+## How the parts work together
+
+SiloHost is started first.
+Then, OrleansClient is started. OrleansClient tells the system to create a grain based on the grain interface provided.
+The grain is then stored in SiloHost. OrleansClient sends a greeting to the grain.
+The grain returns a response to OrleansClient, which OrleansClient displays on the console.
+
+## Step One: Create the project structure
+
+1. In Visual Studio, create a new Visual C# Console App (.NET Core) project.
+2. Name the project **SiloHost** and name the solution **HelloWorld**.
+3. Add a second .NET Core Console App project to the HelloWorld solution and name it **OrleansClient**.
+4. Add another new project, but this time choose .NET Standard – Class Library. Name it **HelloWorld.Interfaces**.
+5. Add a new interface class file to the GrainInterfaces folder and name it **IHello**.
+  *Note: If  Visual Studio adds a default class named `Class1.cs`, feel free to delete this file to keep things tidy.
+6. For the fourth and final project, add another .NET Standard – Class Library project and name it **HelloWorld.Grains**.
+7. Inside the Grains project, add a class and name it **HelloGrain.cs**.
+
+The structure of your Solution should look like this:
+
+```
+
+Solution 'HelloWorld' (4 projects)
```

Comment:
I don't think we need to detail all the dependency and solution structure. We already provided this information in the Sample code. We can just provide a link to the HelloWorld sample, and that would be clear enough IMO.

Reply:
Okay. :)